tests/fast-integration/logging/client.js

```diff
@@ -28,8 +27,20 @@ async function lokiSecretData() {
   return secretData['loki.yaml'];
 }
 
-function getVirtualServices() {
-  return getAllVirtualServices();
+function getLokiVirtualService() {
+  return getVirtualService('kyma-system', 'loki');
+}
+
+async function logsPresentInLoki(query, startTimestamp) {
+  for (let i = 0; i < 20; i++) {
+    const logs = await queryLoki(query, startTimestamp);
+    if (logs.streams.length > 0) {
+      return true;
+    }
+    await sleep(5 * 1000);
+    console.log('sleep');
```

Comment:
Is that log line really meaningful? If yes, please make it more expressive like telling how long you are sleeping in which context

Reply:
No, logging this is not meaningful :D I removed it 👍